src/components/TableHeadCell.js

```diff
@@ -161,7 +165,7 @@ class TableHeadCell extends React.Component {
             </span>
           </Tooltip>
         ) : (
-          <div className={classes.sortAction}>
+          <div className={hint ? classes.sortAction : classes.noSortAction}>
```

Comment:
What if we just do `hint ? classes.sortAction : null`? Then we don't even need replacement styles and it looks like it prevents the pointer cursor from showing.

Reply:
@lalong13 Any interest in submitting that change? If not, I can open up something based on this, just wanted to make sure before using your work or going a different way.

Reply:
@gabrielliwerant Sorry for the delay. I had assumed that display and verticalAlign style properties were still desired. I don't mind not giving it a class name personally. It just might be weird if the other stylings change.

Reply:
I appreciate the cautious approach, but those styles are only needed when the div acts as a container for other things, like sort and hint. They serve no useful function when we don't have a sort or hint.

If some other thing is added here in the future, we can add styles back, but for now, I think it's cleaner to just remove them.

Reply:
I made the suggested change. I can roll the commits up as well if needed/desired.

Reply:
Excellent, will verify soon! What do you mean by "roll the commits up?"

Reply:
Make the two commits into just one commit.

Reply:
Oh, gotcha. No need, I squash them when I merge.

Reply:
@gabrielliwerant Do I need to do anything else for this at this point?

Reply:
Not sure, I need to re-checkout and look everything over, just a bit backed up with PRs and such, but I'll swing around to it before the next minor release.